website/project/views/node.py

```diff
@@ -795,12 +795,12 @@ def _view_project(node, auth, primary=False,
     if embed_registrations:
         data['node']['registrations'] = [
             serialize_node_summary(node=each, auth=auth, show_path=False)
-            for each in node.registrations_all.order_by('-registered_date').exclude(is_deleted=True).annotate(nlogs=Count('logs'))
```

Comment:
Drive-by: The `nlogs` key should also be removed from serialization in `serialize_node_summary`. If you remove this annotation but don't remove the serialization, we'll be doing an expensive log count query for each node.

Reply:
You did this. Disregard.